### PR TITLE
Pyportal ansible

### PR DESCRIPTION
**Pull request by reavdhoef**
Updated the Ansible scripts with:
  - Configurations now put in flask.cfg
  - deployment user no longer root
  - updated .gitignore

### CODE REVIEW

File: roles/yoda_portal/tasks/main.yml [hunk 1]
```diff
@@ -1,22 +1,42 @@
...
+
+
 - name: Checkout Yoda portal
+  become_user: "{{ yoda_deployment_user }}"
+  become: yes
   git:
     repo: "https://github.com/UtrechtUniversity/yoda-portal.git"
     dest: "/var/www/yoda"
-    version: "{{ yoda_portal_version }}"
+    version: "pyportal-ansible"
```
Comment (lwesterhof) [inline roles/yoda_portal/tasks/main.yml]: Should probably be {{ yoda_portal_version }} ?

**Review by lwesterhof — CHANGES_REQUESTED** [review]

File: roles/yoda_portal/tasks/main.yml (hunk 1 above)
Comment (reavdhoef) [reply roles/yoda_portal/tasks/main.yml]: Yes, updated (reverted) it to {{ yoda_portal_version }}

File: roles/yoda_portal/defaults/main.yml [hunk 2]
```diff
@@ -32,6 +29,14 @@ openssl_certs_dir: '/etc/pki/tls/certs'
 openssl_key_signed: localhost.key
 openssl_crt_signed_and_chain: localhost_and_chain.crt
 
+# Flask configuration
+flask_secret_key: PleaseChangeMe123
+flask_portal_title: Yoda Portal
```
Comment (lwesterhof) [inline roles/yoda_portal/defaults/main.yml]: We also have the portal_title_text configuration? Is this redundant?

Comment (reavdhoef) [reply roles/yoda_portal/defaults/main.yml]: Indeed redundant. Replaced with original portal_title_text in other files and made the default 'Yoda Portal', for the development/allinone environment the default is 'Yoda - Dev'.
The other old portal variables need to be checked as well if they can be deleted.

**Pull request merged**